tests/unit/test_time_series.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+import datetime
+import pandas as pd
+
+from cdf_fabric_replicator.time_series import TimeSeriesReplicator
+from cognite.client.data_classes.datapoints_subscriptions import (
+    DatapointsUpdate,
+    Datapoints,
+    DatapointSubscriptionBatch
+)
+from cdf_fabric_replicator.config import SubscriptionsConfig
+from cognite.extractorutils.metrics import BaseMetrics
+from cognite.extractorutils.base import CancellationToken
+import pytest
+from unittest import mock
+from cdf_fabric_replicator.time_series import TimeSeriesReplicator, SubscriptionsConfig, DatapointsUpdate
+from cognite.client import CogniteClient
+
+
+@pytest.fixture
+def mock_token():
+    yield "test_access_token"
+
+
+@pytest.fixture
+def mock_get_token(mocker, mock_token):
+    yield mocker.patch("cdf_fabric_replicator.time_series.TimeSeriesReplicator.get_token", return_value=mock_token)
+
+@pytest.fixture
+def input_data_not_null():
+    yield [DatapointsUpdate(
+            time_series="test_ts",
+            upserts=Datapoints(
+                external_id="id1",
+                timestamp=[1631234567000, 1631234568000],
+                value=[1.23, 4.56]
+            ),
+            deletes=[]
+        )]
+
+@pytest.fixture
+def input_data_batch(input_data_not_null):
```

Comment:
This one is not needed. When removing it, remember to remove the import of `DatapointSubscriptionBatch` too.